highfidelity/hifi
# Move sound effect out of event loop. convert snap.wav from 32bit floa… [DO NOT MERGE]

…t to 16 bit to be supported by QSound

Test plan:
- Make sure notification sounds turned on
- Open Tablet, press Snap
- Hear snapshot sound
